src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -939,6 +928,47 @@ def body(self):
         finally:
             os.remove(config_file)
 
+class VMSSExtensionInstallTest(VCRTestBase):
```

Comment:
by leaving the old test type here it is clearer that the test hasn't been updated to the new pattern yet.  I added issue #741 to clean up the tests.  That said, there are enough of both patterns that both will continue to be used until we do such a cleanup.

---
In reply to: [76134986](https://github.com/Azure/azure-cli/pull/737#discussion_r76134986) [](ancestors = 76134986,76100732,76078126)